README.rst

```diff
@@ -105,6 +105,40 @@ You must specify at least one option, otherwise the entry point does not exist.
 
 ``module``
     Use this when your package name is different from what you import in Python.
+    See also the next section.
+
+
+Different project and module name
+---------------------------------
+
+Usually the project name of an add-on (what is in ``setup.py`` or ``setup.cfg``) is the same as how you would import it in Python code.
+It could be different though.
+In that case, you may get a ``ModuleNotFoundError`` on startup: ``plone.autoinclude`` tries to import the project name and this fails.
+
+The easiest way to solve this, is to switch from ``z3c.autoinclude.plugin`` to ``plone.autoinclude.plugin``, if you have not done so already,
+and specify the module.
+In ``setup.py``::
+
+    setup(
+        name="example.different2",
+        entry_points="""
+        [plone.autoinclude.plugin]
+        module = example.somethingelse2
+        """,
+    )
+
+If you must still support Plone 5.2 and are tied to ``z3c.autoinclude.plugin``, or if you cannot edit the problematic package, you can work around it.
```

Comment:
I'm a bit confused here - how does setting the env var help when using z3.autoinclude?

Reply:
ok got it - initially I misread this to refer to the `z3c.autoinclude` package, not the `z3c.autoinclude.plugin` entry point.